docs/config/cluster.rst

```diff
@@ -714,26 +714,31 @@ Allow to control the allocation of all shards based on include/exclude filters.
 E.g. this could be used to allocate all the new shards on the nodes with
 specific IP addresses or custom attributes.
 
+.. NOTE::
+
+  Applying allocation filters will cause non matching nodes to move off their
+  shards and to the deallocation of the node.
```

Comment:
there's a mismatch in grammatical tense between the two list items

how about:

> If you apply allocation filters, non-matching nodes are deallocated after their shards have been moved off to allocated nodes.